sdks/python/apache_beam/runners/interactive/interactive_runner_test.py

```diff
@@ -147,6 +150,97 @@ def process(self, element):
     ]
     self.assertEqual(actual_reified, expected_reified)
 
+  def test_streaming_wordcount(self):
+    class WordExtractingDoFn(beam.DoFn):
+      def process(self, element):
+        text_line = element.strip()
+        words = text_line.split()
+        return words
+
+    # Add the TestStream so that it can be cached.
+    ib.options.capturable_sources.add(TestStream)
+    ib.options.capture_duration = timedelta(seconds=1)
```

Comment:
Why is this need?

Reply:
Because the background_caching_job is using the real-time clock instead of the fake clock from the runner, we limit the capture duration to 1s to limit the length of the test.